Libraries/LibWeb/HTML/HTMLAnchorElement.cpp

```diff
@@ -122,22 +122,18 @@ void HTMLAnchorElement::activation_behavior(Web::DOM::Event const& event)
 
     // 5. If the user has expressed a preference to download the hyperlink, then set userInvolvement to "browser UI".
     // NOTE: That is, if the user has expressed a specific preference for downloading, this no longer counts as merely "activation".
-    if (has_download_preference())
+    if (has_attribute(HTML::AttributeNames::download))
```

Comment:
It seems to contradict what the comment says. I'm not sure I understand how it is supposed to work, but having download attribute is not the same as "user had expressed a preference"

Reply:
It's OK.